src/test/java/com/dnd/modutime/core/room/integration/RoomTimeTableInitializerTest.java

```diff
@@ -70,4 +70,68 @@ public class RoomTimeTableInitializerTest {
         assertThat(counts).hasSize(2)
                 .allMatch(it -> it.equals(0));
     }
+
+    @Test
+    void 시작시간이_끝시간보다_작을때_해당_정보에_맞게_time_table을_세팅한다() {
+        // given
+        Room room = getRoom(_11_00, _13_00, List.of(_2023_02_09), 2);
+        roomRepository.save(room);
+        TimeTable timeTable = timeTableRepository.save(new TimeTable(room.getUuid()));
+        roomTimeTableInitializer.initialize(room.getUuid(), timeTable);
+
+        // when
+        List<TimeInfo> timeInfos = timeTable.getDateInfos().get(0).getTimeInfos();
+
+        // then
+        assertAll(
+            () -> assertThat(timeInfos.get(0).getTime().equals(_11_00)),
+            () -> assertThat(timeInfos.get(1).getTime().equals(_11_30)),
+            () -> assertThat(timeInfos.get(2).getTime().equals(_12_00)),
+            () -> assertThat(timeInfos.get(3).getTime().equals(_12_30))
+        );
```

Comment:
r: 아무검증도 하지 않고있는데요 ?!

```suggestion
        assertAll(
            () -> assertThat(timeInfos.get(0).getTime().equals(_11_00)).isTrue()
            ...
        );
```

Reply:
isEqualTo 로 썻어야됐는데 그냥 equals로 해버렸네요 😅

Reply:
isEqualTo로 수정해놓겠습니다!